### ISSUE TITLE
Generator [entries]: Fluid fields don't display the names of the fields inside them

### ISSUE THREAD

**Issue by matthewjohns0n** [ExpressionEngine/ExpressionEngine#4516]
DP 4 I think- Robin 10/8

A standalone Grid field: Grid name is 'Video' it has two columns, ID and Type. It is empty.  Output:

![Image](https://github.com/user-attachments/assets/ec53daa4-aa6d-42fd-b62d-0f55c68ab3c5)

Same entry, standalone Grid field: Grid named 'Image', has 4 columns.  It is NOT empty and has 1 row- this is the content:

![Image](https://github.com/user-attachments/assets/ff9a0c05-89af-4a26-a092-aa32f73eba53)

This is how it outputs- note it lost the table and the column headings:

![Image](https://github.com/user-attachments/assets/76ea3d55-6489-454b-be33-47ac1473223f)

### FLUID field###

Fluid field has NO field names showing

Fluid grid- shows the column names for my grid with content and puts it in a table.  Obviously doesn't show anything with no content, which makes sense for a fluid field.  But it still doesn't show the grid name.

![Image](https://github.com/user-attachments/assets/c895d5fc-c829-47fe-9234-4378892eb82d)

------------------------------------

TLDR:
If you have a grid, template code is generated... If you put that same grid in a fluid.. the template code generated for the same grid in a fluid is different then the grid template code generated when it's flying solo. (Might just be name of grid field missing) 

So the name of fields show even when there is no content in the field.  And with a grid, the name and the column names show:

![Image](https://github.com/user-attachments/assets/91a7658f-a163-4040-bb0a-87e5fe77f2a3)

But- In a fluid field, the names of the fields in that fluid field don't show.  Though weirdly, the column names on a grid do show- which was a bit extra confusing.

Here's an example with a fluid field with 2 field groups in it.  The fluid field is named my_fluidg.  You see that, and you see column names from some grids in that fluid field.  But you don't see the field names.

![Image](https://github.com/user-attachments/assets/769b9ca3-2ea9-41b8-bc45-0f2378e9f89b)
